.travis.yml

```diff
@@ -4,9 +4,18 @@ python:
  - "2.7"
  - "3.4"
  - "3.5"
+env:
+ - DJANGO_VERSION=1.8.16
+ - DJANGO_VERSION=1.9.11
+ - DJANGO_VERSION=1.10.4
 install:
-  - pip install tox-travis
```

Comment:
We've agreed in the past that Smartmin will always support last 3 versions of Django, so unless there's a strong argument for supporting 1.7, I don't think we should.